lib/spack/spack/build_systems/cmake.py

```diff
@@ -168,7 +168,9 @@ def _std_args(pkg):
             define('CMAKE_INSTALL_PREFIX', pkg.prefix.replace('\\', '/')),
             define('CMAKE_BUILD_TYPE', build_type),
             define('CMAKE_C_COMPILER:FILEPATH', pkg.compiler.cc.replace('\\', '/')),
-            define('CMAKE_CXX_COMPILER:FILEPATH', pkg.compiler.cxx.replace('\\', '/'))
+            define('CMAKE_CXX_COMPILER:FILEPATH', pkg.compiler.cxx.replace('\\', '/')),
+            define('CMAKE_Fortran_COMPILER:FILEPATH',
+                   pkg.compiler.fc.replace('\\', '/'))
```

Comment:
This is the line that's causing the buildtest to fail because fc is None. Can we check if fc is None and not set `'CMAKE_Fortran_COMPILER` or set it an empty string?

Something like:

```
fortran_compiler = pkg.compiler.fc if pkg.compiler.fc is not None else ''
...
define('CMAKE_Fortran_COMPILER:FILEPATH',  fortran_compiler.replace('\\', '/')
```